components/molecule/dropdownOption/src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import AtomInput from '@s-ui/react-atom-input'
+
+const CLASS = 'sui-MoleculeDropdownOption'
+
+class MoleculeDropdownOption extends Component {
+  state = {
+    selected: this.props.selected
+  }
+
+  handleClick = () => {
+    if (!this.props.disabled) {
+      this.setState(
+        prevState => ({
+          selected: !prevState.selected
+        }),
+        () => {
+          this.props.onClick(this.state.selected)
+        }
+      )
+    }
+  }
+
+  render() {
+    const {selected} = this.state
+    const {children, checkbox, disabled} = this.props
+    const wrapperClassName = cx(CLASS, {
+      [`${CLASS}-checkbox`]: checkbox,
+      [`${CLASS}--disabled`]: disabled,
+      [`${CLASS}--selected`]: selected
```

Comment:
Selected is not a modifier, is a state. For SUIT, it should follow `is-selected`.